docs/kind.md

```diff
@@ -37,12 +101,17 @@ usage: kind.sh [[[-cf|--config-file <file>] [-kt|keep-taint] [-ha|--ha-enabled]
 ```
 As seen above if you do not specify any options script will assume the default values.
 
-You can create your own KIND J2 configuration file if the default one is not sufficient.
+### Usage Notes 
+
+- You can create your own KIND J2 configuration file if the default one is not sufficient
 
-You can also specify these values as environment variables. Command line parameters will override the environment variables.
+- You can also specify these values as environment variables. Command line parameters will override the environment variables.
 
-After deploying the KIND cluster, you can manage the cluster with regular KIND and kubectl commands.
+- To tear down the KIND cluster when finished simply run 
 
+   ```
+   $ kind delete cluster --name ovn
```

Comment:
maybe for now we should add the new delete method:
https://github.com/ovn-org/ovn-kubernetes/pull/1554

Otherwise F31 and F32 users will crash their laptops ;)

Reply:
Definitely I haven't run into that issue but don't want to risk it.